curations/git/github/open-source-parsers/jsoncpp.yaml

```
@@ -31,6 +31,9 @@ revisions:
   d2e6a971f4544c55b8e3b25cf96db266971b778f:
     licensed:
       declared: MIT OR OTHER
+  d84702c9036505d0906b2f3d222ce5498ec65bc6:
+    licensed:
+      declared: MIT
```

The Public Domain clause is probably the `OTHER` referenced in the other versions. Should we do the same here?

```suggestion
      declared: MIT OR OTHER
```